### ISSUE TITLE
Language of society B369 (Norton Sound Inuit)

### ISSUE THREAD

**Issue by KatjaMellmann** [D-PLACE/dplace-cldf#2]
Hi there!
Society B369 (Norton Sound Inuit) is presently linked to a Yupik language (kusk1241). I would assume nort2945 - an Inuit language - would rather be correct here. Or what was your source for this attribution?
Thanks for considering,
Katja

**Comment by xrotwang** [comment]
There are no notes recorded for this decision (see https://github.com/D-PLACE/dplace-dataset-binford/blob/main/raw/societies.csv#L16). So I guess, it must be based on the two sources cited by Binford for this society (see https://d-place.org/society/B369). Both sources - [Seltzer 1933](https://glottolog.org/resource/reference/id/686104) and [Ray 1992](https://glottolog.org/resource/reference/id/685033) are present in Glottolog, but the language coding there is also only based on the D-PLACE data.

We'll keep digging - leaving the issue opened until we find something.

**Comment by KatjaMellmann** [comment]
Sorry for bothering you with that. I've done a few researches myself in the meantime and now think it's ok the way you did it and you don't need to track it down again. Norton Sound has indeed been a Yupik language region. The confusing designation of Norton Sound "Inuit" seems to adhere to a convention in the more outdated literature.

Instead I was wondering why, for several societies (Na6/B299, B296, B369), you put kusk1241 (a dialect) instead of cent2127 (the language level), which at least in the case of the Nunivak (Na6/B299) might be a problematic decision.

But this is just cosmetics. Thanks for your database, btw. I am an enthusiastic user since quite a few years.

**Comment by xrotwang** [comment]
I agree that assigning society Nunivak (Na6/B299) to a sister dialect of the dialect Nunivak seems problematic. And while the Glottolog dialect repository is still quite volatile, Nunivak has been included also at the time the D-PLACE data was initially aggregated. So I'll leave this issue open as a reminder that this might be worth checking.

**Comment by KatjaMellmann** [comment]
This is what I found in Ray 1975: "[...] at the end of the eighteenth century and the beginning of the nineteenth the coastal area from around Solomon to Pastolik was occupied by people who spoke the Unaluk dialect of the Yupik language." (p. 135; see also map 2 on p. 5). "Today most of the names still used by Eskimos on the Norton Sound coast are Unaluk" (ibid., p. 137). So, according to this source, D-PLACE's attribution of a Yupik language to the Norton Sound "Inuit" is absolutely correct. However, the presently chosen dialect kusk1241 (General Central Yupik) should either be shifted to the language level cent2127 (Central Alaskan Yupik) or switched to the dialect unal1234 (Unaliq).
